server/installation.go

```diff
@@ -197,6 +197,14 @@ func (p *Plugin) getInstallation(installationID string) (*Installation, error) {
 
 	for _, install := range installs {
 		if install.ID == installationID {
+			// Retrieve the information we need from the installation directly from the provisioner
```

Comment:
Do we need to check if the installation already has the DNS records? This might save an extra call.

Reply:
I've added a check for when/if we modify the webhook data to send the DNSRecords along, because right now we're not sending it. [ref](https://github.com/fmartingr/mattermost-cloud/blob/master/internal/events/producer.go#L53-L69)